samples/virtio/entropy/boards/xenvm_gicv3.overlay

```diff
@@ -0,0 +1,22 @@
+/{
+	soc {
+		virtio_mmio@a003e00 {
+			dma-coherent;
+			interrupt-parent = <&gic>;
+			interrupts = <0x00 0x2f 0x01 0x00>;
+			reg = <0x00 0xa003e00 0x00 0x200>;
+			compatible = "virtio,mmio";
+			status = "okay";
+			label = "virtio0";
```

Comment:
needed?

Reply:
Exactly why label is used ? I am not sure if it's standard property.

Reply:
I suspect we inherited label properties from Zephyr, since at some point they were used frequently. However, the current recommendation from https://docs.zephyrproject.org/latest/build/dts/intro-syntax-structure.html is:

_Label properties continue to persist for historical reasons in some existing bindings and overlays, but should not be used in new bindings or device implementations._